gradle/libs.versions.toml

```diff
@@ -5,10 +5,10 @@ ktlint = "0.50.0"
 lint = "31.0.2"
 
 [libraries]
-android-core = { group = "androidx.core", name = "core-ktx", version.strictly = "1.10.1" }
-android-lifecycle = { group = "androidx.lifecycle", name = "lifecycle-runtime-ktx", version.strictly = "2.6.1" }
+android-core = "androidx.core:core-ktx:1.10.1"
```

Comment:
It works? :D Why it is stored like this not using `{ group = ..., name = ..., version... = ...`?

Reply:
https://docs.gradle.org/current/userguide/platforms.html#sub::toml-dependencies-format

![image](https://github.com/intive/Picover/assets/12697488/e32e77f7-ba67-425c-9ea8-90247e820e63)

That's why it's always worth to check documentation 😄